src/host/broadcast.jl

```diff
@@ -47,7 +47,15 @@ end
     copyto!(similar(bc, ElType), bc)
 end
 
-@inline function Base.copyto!(dest::BroadcastGPUArray, bc::Broadcasted{Nothing})
+@inline function materialize!(::Style, dest, bc::Broadcasted) where {Style<:AbstractGPUArrayStyle}
```

Comment:
What is this function? It's used nowhere.

Reply:
Opps. It should be `Base.materialize!`. Test passed because we have L54